doc/rule-descriptions.md

```diff
@@ -33,6 +33,7 @@
 | input-image-alt | Ensures &lt;input type=&quot;image&quot;&gt; elements have alternate text | cat.text-alternatives, wcag2a, wcag111, section508, section508.22.a | true |
 | label-title-only | Ensures that every form element is not solely labeled using the title or aria-describedby attributes | cat.forms, best-practice | false |
 | label | Ensures every form element has a label | cat.forms, wcag2a, wcag332, wcag131, section508, section508.22.n | true |
+| landmark-at-least-one-main | Each page must have at least one main landmark, used to identify the main content. | cat.keyboard | true |
```

Comment:
it does and it did, so this change being committed as part of the PR is what we want so when it merges, the documentation in GitHub is updated